ui/src/app/user/user.component.html

```diff
@@ -139,13 +139,11 @@
             </ion-item>
 
             <ion-note class="ion-padding" *ngIf="environment.backend === 'OpenEMS Edge'">
-              <small>
-                The language can not be changed permanently in the local online-monitoring for technical
-                reasons.</small>
-            </ion-note>
+              <small>{{ 'About.info' | translate }}</small>
```

Comment:
Please create new translations in UPPER_CASE, as this is best practice.

Reply:
done :)